### DISCUSSION TITLE
Lorem component

### DISCUSSION THREAD

**Discussion by codams** [chakra-ui/chakra-ui#5676] [Q&A] [answered]
Hi all !

Am i missing something or has the **Lorem component been removed from chakra-ui/react ?**
Many examples in docs [mention it](https://chakra-ui.com/docs/overlay/modal#usage) but i can't import it 😭
<img width="907" alt="image" src="https://user-images.githubusercontent.com/7759894/156395284-7bbdf373-0975-4426-8b87-e8fa12e7b4ca.png">

If it has been removed, can I make a PR to update the docs ?

My env is typescript with `"@chakra-ui/react": "^1.7.2",`

Have a great day

**Comment by BasantaPoudel** [comment]
Hello codams, I am with the similar problem and apparently it seems to have been removed.

**Comment by codams** [comment, marked as answer]
Sooo, Lorem component is not from chakra-ui 👻
If anyone ends up here it's from [this repo](https://github.com/martinandert/react-lorem-component)

**Comment by aScHEABL** [comment]
If the component in question is not from Chakra UI then the docs should be crystal clear about it when it's used as the example.
[Link](https://chakra-ui.com/docs/components/transitions/usage#collapse-transition)
1 year later and I stumble on this issue again.
